components/ws-manager-api/typescript/src/core_pb.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
```

Comment:
I was under the impression we shouldn't update these ([ref](https://gitpod.slack.com/archives/C01KGM9AW4W/p1645090427983519))

Reply:
:100: :point_up: